velox/docs/configs.rst

```diff
@@ -432,6 +432,23 @@ Hive Connector
      -
      - The GCS service account configuration as json string.
 
+``Azure Blob Storage Configuration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. list-table::
+   :widths: 30 10 10 60
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - fs.azure.account.key.<storage-account>.dfs.core.windows.net
+     - string
+     -
+     - The credentials to access the specific Azure Blob Storage account. Replace <storage-account> with the name of your Azure Storage account.
+        This property aligns with how Spark configures Azure account key credentials for accessing Azure storage. 
```

Comment:
The formatting looks off here https://github.com/facebookincubator/velox/blob/14167541de853899a42b0b0ec651b87c9e236dcf/velox/docs/configs.rst
A new line probably requires a [line block](https://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#line-blocks). Can you build the doc locally and verify?

Reply:
make a small change and verified in local , looks should be fine.

![image](https://github.com/facebookincubator/velox/assets/9858245/663ce5b6-6ebb-4a21-82a9-dde4913f6f94)